app/web/features/profile/view/leaveReference/LeaveReferencePage.tsx

```diff
@@ -36,24 +35,27 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function LeaveReferencePage() {
+export default function LeaveReferencePage({
+  referenceType,
+  userId,
+  hostRequestId,
+}: {
+  referenceType: string;
+  userId: number;
+  hostRequestId?: number;
+}) {
   const classes = useStyles();
-  const { referenceType, userId, hostRequestId } = useParams<{
-    referenceType: string;
-    userId: string;
-    hostRequestId?: string;
-  }>();
 
   const {
     data: user,
     isLoading: isUserLoading,
     error: userError,
-  } = useUser(+userId);
+  } = useUser(userId);
   const {
     data: availableRefrences,
```

Comment:
```suggestion
    data: availableReferences,
```
Ever since I installed the spell checker plugin, this typo has been annoying me 😛 so I think we should fix it here